media/css/firefox/privacy/products.scss

```diff
@@ -0,0 +1,298 @@
+                @include bidi(((left, 0, right, 0),));
+                background-color: transparent;
+                background-repeat: no-repeat;
+                margin: 0;
+            }
+        }
+
+        &.invisible .mzp-c-card-picto-content:before {
+            @include background-size(103px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-invisible.svg');
+            height: 80px;
+            width: 103px;
+        }
+
+        &.control .mzp-c-card-picto-content:before {
+            @include background-size(62px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-control.svg');
+            height: 80px;
+            width: 62px;
+        }
+
+        &.protection .mzp-c-card-picto-content:before {
+            @include background-size(94px 80px);
+            background-image: url('/media/img/firefox/privacy/products/icon-protection.svg');
+            height: 80px;
+            width: 94px;
+        }
+
+        @media #{$mq-md} {
+            .mzp-c-card-picto-content:before {
+                @include bidi(((left, $spacing-lg, right, 0),));
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Tracker Counter Call Out
+
+.privacy-products-tracker-counter.mzp-t-dark {
+    background-color: $color-violet-80;
+
+    .mzp-c-call-out-title {
+        margin: 0 auto $spacing-lg;
+        max-width: 24em;
+
+        @media #{$mq-md} {
+            @include font-size(32px);
+        }
+
+        @media #{$mq-lg} {
+            @include font-size(40px);
+        }
+    }
+
+    .mzp-c-cta-link {
+        color: $color-blue-30;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Product List Feature Cards
+
+.privacy-products-list {
+    margin: $layout-md auto;
+
+    .mzp-c-card-feature {
+        margin-bottom: $layout-xl;
+
+        &.has-logo {
+            .mzp-c-card-feature-title {
+                @include bidi(((background-position, top left, top right),));
+                @include image-replaced;
+                background-repeat: no-repeat;
+                margin-bottom: $spacing-lg;
+                min-height: 40px;
+            }
+        }
+
+        &:last-child {
+            margin-bottom: 0;
+        }
+    }
+
+    .mzp-c-card-feature-desc {
+        margin-bottom: $spacing-xl;
+    }
+
+    .mzp-c-card-feature-media img {
+        max-width: 320px;
+    }
+
+    @media #{$mq-lg} {
+        margin: $layout-xl auto;
+
+        .mzp-c-card-feature-media img {
+            max-width: 410px;
+        }
+    }
+}
+
+.privacy-products-monitor .mzp-c-card-feature-title {
+    @include at2x('/media/protocol/img/logos/firefox/monitor/logo-word-hor-sm.png', auto, 40px);
+}
+
+.privacy-products-lockwise .mzp-c-card-feature-title {
+    @include at2x('/media/protocol/img/logos/firefox/lockwise/logo-word-hor-md.png', auto, 40px);
```

Comment:
I think you can use the `-sm` size for the monitor and send like you do the monitor one.